1 Presentation layer/CandidateEvaluator.Server/Startup.cs

```diff
@@ -92,14 +92,14 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<ICommandHandler<CreateInterviewResult>, CreateInterviewResultHandler>();
             services.AddTransient<ICommandHandler<DeleteInterviewResult>, DeleteInterviewResultHandler>();
 
-            services.AddTransient<IQueryHandler<GetAllCategories, List<Category>>, GetAllCategoriesHandler>();
+            services.AddTransient<IQueryHandler<GetAllCategories, IEnumerable<Category>>, GetAllCategoriesHandler>();
```

Comment:
why IEnumerable? I think ICollection will be more useful than IEnumerable

Reply:
ICollection is mutable